packages/firestore/src/remote/serializer.ts

```diff
@@ -975,6 +975,11 @@ export function toTarget(
 
   if (targetData.resumeToken.approximateByteSize() > 0) {
     result.resumeToken = toBytes(serializer, targetData.resumeToken);
+  } else if (targetData.snapshotVersion.compareTo(SnapshotVersion.min()) > 0) {
```

Comment:
Can we:
- Always update the read time for each TargetData?
- Always persist it?